include/oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h

```diff
@@ -52,7 +52,7 @@ class all_view
     return_t*
     begin() const
     {
-        return m_acc.get_pointer();
+        return &m_acc[0];
```

Comment:
can we write utility function for that to do not duplicate workaround in three places?

Reply:
Yes, but it doesn't make sense, because after the test refactoring will remain just one place (utils_ranges_sycl.h).